src/app/vm/vm-creation/vm-creation.component.html

```diff
@@ -153,11 +153,12 @@ <h3 class="mat-dialog-title">{{ 'VM_PAGE.VM_CREATION.NEW_VM' | translate }}</h3>
           </div>
 
           <div>
-            <h5>{{ 'VM_PAGE.VM_CREATION.AFFINITY_GROUPS' | translate }}</h5>
+            <h5>{{ 'VM_PAGE.VM_CREATION.AFFINITY_GROUP' | translate }}</h5>
             <mat-form-field floatPlaceholder="never">
               <input
                 name="group"
                 matInput
+                pattern="[a-zA-Z]{1}[a-zA-Z0-9\-\_]{0,61}[a-zA-Z0-9]$"
```

Comment:
Can we add a max length option to help user to understand why if they enter long affinity group name, save button is disabled?